retail/interactive-tutorial/RetailEvents.Samples.Tests/PurgeUserEventTest.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Copyright 2022 Google LLC. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Cloud.Retail.V2;
+using System;
+using Xunit;
+
+namespace RetailEvents.Samples.Tests
+{
+    public class PurgeUserEventTest
+    {
+        [Fact]
+        public void TestPurgeUserEvent()
+        {
+            string projectId = Environment.GetEnvironmentVariable("GOOGLE_PROJECT_ID");
+            string projectNumber = Environment.GetEnvironmentVariable("PROJECT_NUMBER");
+
+            string defaultCatalog = $"projects/{projectId}/locations/global/catalogs/default_catalog";
+            string expectedPurgeResponseName = $"projects/{projectNumber}/locations/global/catalogs/default_catalog/operations/purge-user-events";
```

Comment:
Same comment as above for this one. And everywhere you find yourselg building these strings.

Reply:
I will adjust all Branches/Catalogs creation in a separate PR by your suggestion